web_src/css/base.css

```diff
@@ -1471,6 +1471,10 @@ img.ui.avatar,
   box-shadow: none;
 }
 
+.ui.vertical.menu .item > .menu {
+  margin-left: 0;
```

Comment:
I would suggest not overwriting other framework's styles (there are already a lot of such technical debts in Gitea's code base).

Reply:
Used `helper.css` instead https://github.com/go-gitea/gitea/pull/24200/commits/d9c776ff4e0ff2917f5d499e15afa1acb6e53990